pkgs/applications/science/biology/cd-hit/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, fetchFromGitHub, zlib, perl }:
+
+stdenv.mkDerivation rec {
+  version = "4.8.1";
+  pname = "cd-hit";
+
+  src = fetchFromGitHub {
+    owner = "weizhongli";
+    repo = pname;
```

Comment:
The repo name is "cdhit" not "cd-hit".

Reply:
Ok. Now, I now that I have to run the garbage-collector before doing a final check, because such an error could not be detected as the sources were already into my store...